apps/api/src/schema/base/response.ts

```diff
@@ -11,11 +11,8 @@ export const cursorResponseSchema = <T extends z.ZodType>(data: T) =>
     ok: z.literal<boolean>(true).openapi({}),
     data: z.object({
       items: data.openapi({ description: "The list of requested items" }),
-      nextCursor: z
-        .string()
-        .nullable()
-        .openapi({
-          description: "Cursor pointing to the next page. Null if there are no more results",
-        }),
+      nextCursor: z.string().nullable().openapi({
+        description: "Cursor pointing to the next page. Null if there are no more results",
+      }),
```

Comment:
This reformatting is out of scope for this PR. Not sure if deliberate or if Biome somehow didn't catch it before.

Reply:
Biome did this. It really is checked in misformatted right now.